pkg/apis/submariner.io/v1/types.go

```diff
@@ -283,6 +283,7 @@ type ClusterGlobalEgressIPList struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:shortName="giip"
+// +kubebuilder:printcolumn:JSONPath=".status.allocatedIP",name="IP",description="Global IP Allocated",type="string"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:JSONPath=".status.allocatedIP",name="Global IP",description="Global IP Allocated",type="string"
```

Just a thought: Shall we use `Global IP`? I'm fine even with IP too.